testing/src/main/graphql/betterbotz/async/BulkInsertProductsWithAsync.graphql

```diff
@@ -0,0 +1,16 @@
+mutation BulkInsertProductsWithAsync(
```

Comment:
These tests should probably make use of @olim7t change in https://github.com/stargate/stargate/pull/906

Reply:
@olim7t, do you want me to wait until your PR is merged, or can I merge and you will adapt yours?

Reply:
Whoops, I just merged it 😬  So the former...

Reply:
Ok, I've adapted to the new IT testing approach.